src/holofuel/components/CopyToClipboard/CopyToClipboard.test.js

```diff
@@ -0,0 +1,63 @@
+import React, { createContext } from 'react'
+import Modal from 'react-modal'
+import { render, fireEvent, act } from '@testing-library/react'
+import wait from 'waait'
+import { Router } from 'react-router-dom'
+import { createMemoryHistory } from 'history'
+import FlashMessage from 'holofuel/components/FlashMessage'
+import CopyToClipboard from './CopyToClipboard.js'
+
+jest.mock('holofuel/components/CopyToClipboard')
+jest.mock('holofuel/components/layout/PrimaryLayout')
+jest.mock('holofuel/contexts/useFlashMessageContext')
+
+const FlashMessageContext = createContext()
+const MockFlashContextProvider = ({ message, time = 5000, newMessage = jest.fn(), children }) => (
+  <FlashMessageContext.Provider value={{ message, time, newMessage }}>
+    {children}
+  </FlashMessageContext.Provider>
+)
+
+const renderWithRouter = (
+  ui,
+  {
+    route = '/',
+    history = createMemoryHistory({ initialEntries: [route] })
+  } = {}
+) => ({
+  ...render(
+    <Router history={history}>
+      <MockFlashContextProvider>
+        <FlashMessage />
+        {ui}
+      </MockFlashContextProvider>
+    </Router>
+  ),
+  history
+})
+
+it('should copy the HolofuelUser Agent Hash to clipboard and display flash message feedback', async () => {
```

Comment:
let's rename the test to something more generic to illustrate that this component is generic.
`it('should copy the copyContent to clipboard and display flash message feedback')`